docs/rfcs/2022-12-17-user-profiles.md

```diff
@@ -0,0 +1,130 @@
+# User profiles
+
+## Problem
+
+Convenience of having different communication contexts in a single client without shared metadata.
+
+Currently requires changing chat database and restarting chat.
+
+Events (notifications) for databases other than current are not received.
+
+## Solution
+
+Support for multiple user profiles in a single database.
+
+Separate transport connections with the same servers not shared between profiles.
+
+## Design
+
+### API
+
+- new users are created using existing CreateActiveUser API
+
+- APIListUsers
+
+- APISetActiveUser UserId
+
+- CRUsersList {users :: [User]}
+
+- terminal APIs
+
+### Subscriptions
+
+- get connections for all users, subscribe to all - to receive events for users other than active
+
+- map subscriptions to users?
+
+- option to subscribe to connections for only one user? `StartChat {allUsers :: Bool}` API
+
+- if more than one user profiles - start with last active user (persist?), or always load list of users first and wait for choice? same for terminal?
```

Comment:
last active makes sense, I don't like the UI when the first the user see is the list of profiles. It should be clear which profile is currently active - avatar should be visible in the list of chats, we would probably need some equivalent if avatar is not set.